src/renderTemplate.js

```diff
@@ -0,0 +1,12 @@
+const RenderPosition = {
+  BEFOREBEGIN: 'beforebegin',
+  AFTERBEGIN: 'afterbegin',
+  BEFOREEND: 'beforeend',
+  AFTEREND: 'afterend',
+};
+
+const renderTemplate = (container, template, place) => {
```

Comment:
```suggestion
const renderTemplate = (container, template, place = RenderPosition.BEFORE_END) => {
``` 

пусть  RenderPosition.BEFORE_END будет значением по умолчанию, что позволит не передавать третий параметр, когда нужен вставить в beforeend

Reply:
Исправил